### PR TITLE
feat: add support for multiple insert in one query

### PR DESCRIPTION
**Pull request by samarpanB**
Added support for bulk insert in a single query using newly introduced multiInsertSupported flag of connector

BREAKING CHANGE
drop nodejs v12 support and juggler v3 support

Signed-off-by: Samarpan Bhattacharya [this.is.samy@gmail.com](mailto:this.is.samy@gmail.com)

Depends on https://github.com/loopbackio/loopback-connector/pull/265

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [x] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

**Review by dhmlau — APPROVED** [review]
@samarpanB, thanks for the PR. LGTM. Please make sure we have at least one more approval before merging. 
I've also added the `breaking-change` label.

File: .github/workflows/ci.yml
Comment (raymondfeng) [inline .github/workflows/ci.yml:18]: Do we want to add `18`?

Comment (samarpan-b) [reply .github/workflows/ci.yml:18]: Sure. I can add it

Comment (samarpanB) [reply .github/workflows/ci.yml:18]: So I checked this. nodejs18 is not supported in ubuntu-18 which is the OS env here. And if I upgrade ubuntu version then the [setup-mysql action](https://github.com/ankane/setup-mysql) fails as it doesn't support mysql5.7 in higher ubuntu versions.
So to add support for node v18, we will need to first support mysql v8 and then update the workflow to have a conditional run where, it should run on mysql v5.7 for node versions 14 & 16. And for v18, it should run on mysql v8. I am creating a separate issue for it and will do it there.
@raymondfeng I hope that works for this PR.

Comment (raymondfeng) [reply .github/workflows/ci.yml:18]: I'm fine not to enable 18 for now.

Comment (samarpanB) [reply .github/workflows/ci.yml:18]: Then I am going ahead to merge it @raymondfeng @dhmlau

**Pull request merged**
